packages/flutter/test/widgets/scrollbar_test.dart

```diff
@@ -1482,4 +1482,54 @@ void main() {
         ..rect(rect: const Rect.fromLTRB(794.0, 10.0, 800.0, 358.0))
     );
   });
+  testWidgets('minThumbLength property of RawScrollbar is respected', (WidgetTester tester) async {
```

Comment:
Can you add a test that verifies the minOverscrollLength?

Reply:
I'll get right on it!